app/assets/stylesheets/govuk_publishing_components/components/helpers/_brand-colours.scss

```diff
@@ -90,3 +90,16 @@
     border-color: govuk-colour("black");
   }
 }
+
+// new brand colour to match Public Health England
+// used on the coronavirus page
+// note that PHE already has a different brand colour, the brand
+// department-for-health is used on the PHE org page
+$covid-green: #7eff8e;
```

Comment:
Would be great to try and keep variables with a `$gem-` prefix

Reply:
Good point, will amend.

Reply:
Thank you!